notes-to-self/socket-scaling.py

```diff
@@ -32,7 +32,7 @@ def pt(desc, *, count=total, item="socket"):
             now = time.perf_counter()
             total_ms = (now - last_time) * 1000
             per_us = total_ms * 1000 / count
-            print(f"{desc}: {total_ms:.2f} ms total, {per_us:.2f} µs/{item}")
+            print(f"{desc}: {total_ms:.2f} ms total, {per_us:.2f} μs/{item}")
```

Comment:
what did this change?

Reply:
Ahh, Ruff changed "U+00B5 MICRO SIGN" to "U+03BC GREEK SMALL LETTER MU". They look identical, but apparently the latter is the preferred encoding to use. According to section 2.5 of [this unicode doc](https://www.unicode.org/reports/tr25/#page11) at least.

Reply:
Correct